packages/clerk-js/src/ui/components/OrganizationProfile/OrganizationMembers.tsx

```diff
@@ -21,15 +21,19 @@ import { OrganizationMembersTabRequests } from './OrganizationMembersTabRequests
 export const OrganizationMembers = withCardStateProvider(() => {
   const { organizationSettings } = useEnvironment();
   const card = useCardState();
-  const { membership } = useCoreOrganization();
-  const isAdmin = membership?.role === 'admin';
-  const allowRequests = organizationSettings?.domains?.enabled && isAdmin;
+  const { isAuthorizedUser: canManageMemberships } = useGate({ permission: 'org:memberships:manage' });
+  const isDomainsEnabled = organizationSettings?.domains?.enabled;
   const { membershipRequests } = useCoreOrganization({
-    membershipRequests: allowRequests || undefined,
+    membershipRequests: isDomainsEnabled || undefined,
   });
-  //@ts-expect-error
+
+  // @ts-expect-error
   const { __unstable_manageBillingUrl } = useOrganizationProfileContext();
 
+  if (canManageMemberships === null) {
```

Comment:
Maybe unrelated to these changes, i would expect this to be a boolean instead every time

Reply:
That is true, but due to the underlying mechanism i think it is useful to know if the method has even resolved yet.

This is particularly useful for tests in order to know if an element does not exist because the method has not resolved yet, or because it has resolved as false